Sources/Models/User.swift

```diff
@@ -28,7 +29,8 @@ public struct User: Decodable, Equatable {
     referralCode: ReferralCode,
     referrerId: Id?,
     rssSalt: RssSalt,
-    subscriptionId: Subscription.Id?
+    subscriptionId: Subscription.Id?,
+    teamInviteCode: TeamInviteCode
```

Comment:
Should the invite code be on the user or on the sub?

Reply:
On the user right? Why would it be on the sub?